### ISSUE TITLE
Alt text not preserved when converting a Markdown image to a Block

### ISSUE THREAD

**Issue by bgreenlee** [TypeCellOS/BlockNote#1882]
**Describe the bug**

Given the following Markdown:

```
![Sample image](https://picsum.photos/200)
```

The "Sample image" is the alt text and should be preserved as such. However the Block HTML for this looks like:

```
<img class="bn-visual-media" src="https://picsum.photos/200" alt="BlockNote image" contenteditable="false" draggable="false">
```

**To Reproduce**

This can be reproduced in the [BlockNote Playground](https://blocknote-main.vercel.app/interoperability/converting-blocks-from-md)
